### ISSUE TITLE
[Bug] PyPI publish action does not work 

### ISSUE THREAD

**Issue by erogol** [coqui-ai/TTS#1519]
### Describe the bug

Check https://github.com/coqui-ai/TTS/runs/6092850161?check_suite_focus=true

### To Reproduce

Check https://github.com/coqui-ai/TTS/runs/6092850161?check_suite_focus=true

### Environment

```shell
Check https://github.com/coqui-ai/TTS/runs/6092850161?check_suite_focus=true
```

**Comment by erogol** [comment]
I published v0.6.2 manually btw

**Comment by erogol** [comment]
@WeberJulian can you check this one?
